examples/gno.land/p/demo/memeland/memeland.gno

```diff
@@ -26,15 +26,15 @@ type Post struct {
 }
 
 type Memeland struct {
-	*ownable.Ownable
+	*ownable.DefaultOwnable
```

Comment:
```suggestion
	ownable.Ownable
```
Should embed the interface, not a specific implem.

Reply:
done